index.html

```diff
@@ -25,131 +25,147 @@
-                        <div class="action1"><button type="submit" id="button" class="button_f">See Project</button></div>  
-                    </div>
-                    </div>
-                </div>
+    <section id="portfolio" class="content">
+      <div class="primary_txt"><h1 class="top_color">Hey there. I'm Kiplaacollo <br><span class="bottom_color"> I'm a software developer</span></h1></div>
+      <div class="supportive_txt">
+        <p> I can help you build a product , feature or website Look through some <br>of my work and experience! If you like what you see and have a project you <br>need coded, don’t hestiate to contact me.</p>
+      </div>
+      <div class="social_icon_container">
+        <ul>
+          <li><a href="#" class="twitter"><img src="images/twitter.png" alt="Twiter login"></a></li>
+          <li><a href="#" class="linkedin"><img src="images/Linkedin icon.png" alt="Linkedin login"></a></li>
+          <li><a href="#" class="medium"><img src="images/medium.png" alt="medium login"></a></li>
+          <li><a href="#" class="github"><img src="images/github.png" alt="github login"></a></li>
+          <li><a href="#" class="angellist"><img src="images/angellist.png" alt="angellist login"></a></li> 
+        </ul>   
+      </div>
+      <div class="action"><img class="normal_button" src="images/Disabled.png" alt="Scroll Bar"></div>   
+    </section>
+    <section class="recent_works">
+      <div id="popup-window" class="head">
+        <div id="overlay">
+        <div id="card-works-container">
+          <span class="close" id="close">&times;</span>
+          <div id="projectImage"></div>
+          <div id="projectTitle"></div>
+          <div id="tags" class="tag"></div>
+          <div id="projectDescription"></div>
+          <button id="actionBtn" class="button_f btn">See live <img src="./images/Icon - Export.svg" alt=""></button>
+          <button id="actionBtn1" class="button_f btn">See Source <img src="./images/IconGitHub.svg" alt=""></button>
+        </div>
+      </div>
+    </div>
+      <div class="primary_txt"><h2>My Recent Works</h2></div>
+      <img class="indicator" src="images/Indicator.png" alt="Indicator">
+      <div class="card_Work_Row_1">
+        <div class="card">
+          
+          <div class="image_placeholder_card"> 
+            <img src="images/projet1.png" alt="" id="mobile-view-img">
+            
+            <div class="text">
+              <div><h4>Multi-Post Stories Gain+Glory</h4></div>
+              <div>
+                <ul class="tag">
+                  <li class="tag1">Ruby on rails</li>
+                  <li class="tag2">css</li>
+                  <li class="tag3">JavScript </li>
+                  <li class="tag4">html</li>
+                </ul>                                                  
+              </div>
+              <div class="action1"><button type="submit" id="0" class="button_f btn">See Project</button></div>  
```

Comment:
- [ ] As mentioned by the previous reviewer, all the project under my recent work section would be created from array of objects in your js file. Already created code for this section in your index.html file should be removed but created from js file. I would recommend you to follow the suggestions by the previous reviewer to accomplish this.